numpy/lib/_function_base_impl.py

```diff
@@ -4400,139 +4267,82 @@ def quantile(a,
+    The table above includes only the estimators from H&F that are continuous
+    functions of probability `q` (estimators 4-9). NumPy also provides the
+    three discontinuous estimators from H&F (estimators 1-3), where ``j`` is
+    defined as above and ``m`` and ``g`` are defined as follows.
 
-        * alpha = 0
-        * beta = 1
+    1. ``inverted_cdf``: ``m = 0`` and ``g = int(q*n > 0)``
+    2. ``averaged_inverted_cdf``: ``m = 0`` and ``g = (1 + int(q*n > 0)) / 2``
+    3. ``closest_observation``: ``m = -1/2`` and
+       ``1 - int((g == 0) & (j%2 == 0))``
 
-    hazen:
-        method 5 of H&F [1]_.
-        This method gives continuous results using:
+    For backward compatibility with previous versions of NumPy, `quantile`
+    provides four additional discontinuous estimators. Like
+    ``method='linear'``, all have ``m = 1 - q`` so that ``j = q*(n-1) // 1``,
+    but ``g`` is defined as follows.
 
-        * alpha = 1/2
-        * beta = 1/2
+    - ``lower``: ``g = 0``
+    - ``midpoint``: ``g = 0.5``
+    - ``higher``: ``g = 1``
+    - ``nearest``: ``g = (q*(n-1) % 1) > 0.5``
 
-    weibull:
-        method 6 of H&F [1]_.
-        This method gives continuous results using:
-
-        * alpha = 0
-        * beta = 0
-
-    linear:
-        method 7 of H&F [1]_.
-        This method gives continuous results using:
-
-        * alpha = 1
-        * beta = 1
-
-    median_unbiased:
-        method 8 of H&F [1]_.
-        This method is probably the best method if the sample
-        distribution function is unknown (see reference).
-        This method gives continuous results using:
-
-        * alpha = 1/3
-        * beta = 1/3
-
-    normal_unbiased:
-        method 9 of H&F [1]_.
-        This method is probably the best method if the sample
-        distribution function is known to be normal.
-        This method gives continuous results using:
-
-        * alpha = 3/8
-        * beta = 3/8
-
-    lower:
-        NumPy method kept for backwards compatibility.
-        Takes ``i`` as the interpolation point.
-
-    higher:
-        NumPy method kept for backwards compatibility.
-        Takes ``j`` as the interpolation point.
+    **Weighted quantiles:**
+    More formally, the quantile at probability level :math:`q` of a cumulative
+    distribution function :math:`F(y)=P(Y \\leq y)` with probability measure
+    :math:`P` is defined as any number :math:`x` that fulfills the
+    *coverage conditions*
 
-    nearest:
-        NumPy method kept for backwards compatibility.
-        Takes ``i`` or ``j``, whichever is nearest.
+    .. math:: P(Y < x) \\leq q \\quad\\text{and}\\quad P(Y \\leq x) \\geq q
 
-    midpoint:
-        NumPy method kept for backwards compatibility.
-        Uses ``(i + j) / 2``.
+    with random variable :math:`Y\\sim P`.
+    Some of the quantile estimators above (namely ``inverted_cdf`` and
+    ``averaged_inverted_cdf``) arise when one considers :math:`F` as the
```

Comment:
```suggestion
    Some of the quantile estimators above, namely ``inverted_cdf`` and
    ``averaged_inverted_cdf``, arise when one considers :math:`F` as the
```

Reply:
The most objective correction here would be to add a comma _after_ "namely". Use of parentheses or commas to surround such a phrase are both valid, as far as I can tell (e.g., https://english.stackexchange.com/questions/145452/punctuation-around-the-word-namely).

Reply:
I suggested it, on particular because I thought you don’t like parentheses in a sentence.

Reply:
https://github.com/numpy/numpy/pull/25704/files#r1590014290

Reply:
```suggestion
    Some estimators arise when one considers :math:`F` as the empirical
```